client/js/app/utils/ExplorerUtils.js

```diff
@@ -265,6 +265,10 @@ module.exports = {
       delete attrs['steps'];
     }
 
+    if (attrs.group_by) {
```

Comment:
Hmmmm, actually I think this may be a problem. We only want to stringify the group_by property when it's an array, otherwise we just want to encodeURIComponent without stringifying it. So this should be: `if (attrs.group_by && _.isArray(attrs.group_by) && attrs.group_by.length > 1)`

We could potentially send the API a single item array stringified, but it feels a bit weird to have people using API urls that have single item arrays stringified. It might mislead them into thinking it *has* to be that way, when in fact for a single group by you can just do standard url encoding.

Does that make sense? I realize this all might be confusing haha.